app/assets/stylesheets/custom.css.scss

```diff
@@ -39,4 +39,42 @@ body {
 
 #sticky-footer {
   flex-shrink: none;
-}
\ No newline at end of file
+}
+
+.gravatar {
+  margin: 5px;
+}
+
+/* Pagination styling */
+.flickr_pagination {
+  text-align: center;
+  padding: 0.3em;
+  cursor: default; }
+  .flickr_pagination a, .flickr_pagination span, .flickr_pagination em {
+    padding: 0.2em 0.5em; 
+    border-radius: 5px;
+    color: black;
+  }
+  .flickr_pagination .disabled {
+    color: grey; }
+  .flickr_pagination .current {
+    font-style: normal;
+    font-weight: bold;
+    color: black; }
```

Comment:
Nur als Information: Es gilt weithin als best-practice css-attribute alphabetisch zu sortieren. Ein Linter würde das in der Regel auch bemängeln.
Im VSCode: Zeilen markieren; <ctrl>+<shift>+p; `sort lines ascending`